src/components/Navbar.tsx

```diff
@@ -0,0 +1,18 @@
+import Link from 'next/link';
+import React from 'react';
+import SigninButton from './SigninButton';
+import { AppBar } from '@mui/material';
+
+// Navigation bar with 3 links and Signin Button
+const Navbar = () => {
+  return (
+    <AppBar position="static" style={{ backgroundColor: 'orange' }}>
+      <Link href={'/'}>Home</Link>
+      <Link href={'/events'}>Events</Link>
+      <Link href={'/volunteers'}>Volunteers</Link>
+      <SigninButton />
+    </AppBar>
```

Comment:
Can the links be wrapped in basic Typography components (no need to specify a variant)? We want to have most all of our text inside of those for consistent fonts. Use the `color="inherit"` prop on the links to preserve the font color.

Also, no need to specify a background color at this point; the designers will define our color scheme which will likely mirror CM's existing color palette